### PR TITLE
[CLOUDCRAFT-457] Crafting Better Diagrams: Cloudcraft's Live Diagramming and Filtering

### PR DESCRIPTION
**Pull request by jamesponddotco**
### What does this PR do? What is the motivation?
Add the beginning of an article about the new live experience in Cloudcraft. Still need to work on an introduction and conclusion, so the article is here mostly for discussion.

### Merge instructions
Please wait until the Cloudcraft team has reviewed the content before going ahead with the review and merging of the article.

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/cloudcraft/getting-started/crafting-better-diagrams.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+If you're a new user, you may already have the new live experience enabled by default.
+
+## Select account and region
+
+Click the dropdown under the **Account** section and select the account you want to scan. If you only added one AWS account to Cloudcraft, it's automatically selected for you.
+
+{{< img src="cloudcraft/getting-started/crafting-better-diagrams/select-account.png" alt="Screenshot highlighting the dropdown to select an account to want to scan in the Cloudcraft interface with a red arrow pointing to the dropdown." responsive="true" style="width:100%;">}}
+
+Under **Region**, select the regions you want to scan. By default, `Global` and `US East (N. Virginia)` are selected, but you can click the **More** button to select or search for additional regions.
```
Comment (licaroline) [inline content/en/cloudcraft/getting-started/crafting-better-diagrams.md:31]: We select `Global` and whichever region they configured as their default region by default, with `US East (N. Virginia)` being the fallback region if they don't have any default region configured. Similarly, Azure regions selected by default are `Global` and their configured default region, or `East US`

File: content/en/cloudcraft/getting-started/crafting-better-diagrams.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+{{< img src="cloudcraft/getting-started/crafting-better-diagrams/select-region.png" alt="Screenshot highlighting the regions to scan in the Cloudcraft interface with red arrows pointing to the regions and the More button." responsive="true" style="width:100%;">}}
+
+After selecting, regions are scanned automatically and the number of resources found is displayed right next to the region name. You can click the **Sync** button above the **Region** section to trigger a manual scan of all selected regions.
+
+## Filter resources
+
+You can filter resources by type and tags.
+
+Tags are automatically detected from your AWS account and displayed in the **Custom tags**, **AWS tags**, and **Kubernetes tags** sections.
```
Comment (licaroline) [inline content/en/cloudcraft/getting-started/crafting-better-diagrams.md:41]: There is also a "Terraform tags" section, which are tags automatically added to resources by Terraform

**Comment by jamesponddotco** [comment]
@licaroline Added an introduction, conclusion, and a few more pieces here and there. Please review when you have time :)

**Comment by maycmlee** [comment]
Created [DOCS-7581](https://datadoghq.atlassian.net/browse/DOCS-7581) for docs review when this is ready.

**Comment by jamesponddotco** [comment]
@maycmlee Content itself should be ready, so you can go ahead and review, if you have the time :)

**Comment by licaroline** [comment]
Should the second video be cropped more? It is showing selection of EC2 and RDS in the beginning, which is shown in the first video

**Comment by jamesponddotco** [comment]
> Should the second video be cropped more? It is showing selection of EC2 and RDS in the beginning, which is shown in the first video

I left it in its entirety in the case the user didn't watch the first one.

**Review by licaroline — APPROVED** [review]

File: content/en/cloudcraft/getting-started/create-your-first-cloudcraft-diagram.md [hunk 3]
```diff
@@ -5,6 +5,8 @@ kind: guide
 
 Cloudcraft allows you to import your AWS cloud environment as *live diagrams*. By reverse-engineering the architecture in your AWS account, Cloudcraft can autogenerate new diagrams or enhance existing ones, saving you hours or even days of work.
 
+<div class="alert alert-info">If you are using Cloudcraft's new live experience, please read this article instead: <a href="https://docs.datadoghq.com/cloudcraft/getting-started/crafting-better-diagrams/" title="Crafting Better Diagrams: Cloudcraft's Live Diagramming and Filtering">Crafting Better Diagrams: Cloudcraft's Live Diagramming and Filtering</a>.</div>
```
Comment (estherk15) [inline content/en/cloudcraft/getting-started/create-your-first-cloudcraft-diagram.md:8]:
```suggestion
<div class="alert alert-info">If you are using Cloudcraft's New Live Experience, see this documentation: <a href="https://docs.datadoghq.com/cloudcraft/getting-started/crafting-better-diagrams/" title="Crafting Better Diagrams: Cloudcraft's Live Diagramming and Filtering">Crafting Better Diagrams: Cloudcraft's Live Diagramming and Filtering</a>.</div>
```

File: content/en/cloudcraft/getting-started/crafting-better-diagrams.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+Click **Apply layout** to create a diagram showing only the selected resources.
+
+Say you want to create a diagram showing only EC2 instances and RDS databases without the `Environment` tag. Click the **Resource** section, deselect all resource types, and select only `EC2` and `RDS`. Then click the **Custom tags** section, click the **Environment** tag, and leave only the `Untagged` option selected.
+
+{{< img src="cloudcraft/getting-started/crafting-better-diagrams/select-specific-resources-and-tags.mp4" alt="A 15 seconds video showing a Cloudcraft user selecting EC2 and RDS instances and untagged resources from the Resource and Custom tags sections." video="true">}}
+
+Click **Apply layout** to create a diagram showing only the selected resources without the `Environment` tag.
+
+<hr>
```
Comment (estherk15) [inline content/en/cloudcraft/getting-started/crafting-better-diagrams.md:72]:
```suggestion
## Feedback
```

File: content/en/cloudcraft/getting-started/crafting-better-diagrams.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+Cloudcraft's new live experience is part of an ongoing effort to improve the user experience and make cloud infrastructure diagramming more efficient and effective. We're excited to see how you use these new features and to [hear your feedback][3].
```
Comment (estherk15) [inline content/en/cloudcraft/getting-started/crafting-better-diagrams.md:74]:
```suggestion
Cloudcraft's New Live Experience is part of an ongoing effort to improve the user experience and make cloud infrastructure diagramming more efficient and effective. Tell us how you use these new features and [give feedback in this form][3].
```

File: content/en/cloudcraft/getting-started/crafting-better-diagrams.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+The same goes for tags. Click the **Custom tags**, **AWS tags**, **Terraform tags**, or **Kubernetes tags** section and select the tags you want to filter by. By default, all tags are selected and displayed in order of the number of resources found, with `Untagged` always at the bottom.
+
+<div class="alert alert-info">Consider focusing on the most relevant resoure types and tags for your specific use case to ensure optimal performance and readability of your diagram.</div>
+
+Say you want to create a diagram showing only EC2 instances and RDS databases. Click the **Resource** section, deselect all resource types, and select only `EC2` and `RDS`.
+
+{{< img src="cloudcraft/getting-started/crafting-better-diagrams/select-specific-resources.mp4" alt="A 9 seconds video showing a Cloudcraft user selecting EC2 and RDS instances from the Resource section." video="true">}}
+
+Click **Apply layout** to create a diagram showing only the selected resources.
```
Comment (estherk15) [inline content/en/cloudcraft/getting-started/crafting-better-diagrams.md:64]:
```suggestion
```suggestion

## Use cases

### Create a diagram showing only EC2 instances and RDS databases

1. Click the **Resource** section.
1. Deselect all resource types, and select **EC2** and **RDS**.
1. Click **Apply layout** to create a diagram showing only the selected resources.

{{< img src="cloudcraft/getting-started/crafting-better-diagrams/select-specific-resources.mp4" alt="A 9 seconds video showing a Cloudcraft user selecting EC2 and RDS instances from the Resource section." video="true">}}
```

File: content/en/cloudcraft/getting-started/crafting-better-diagrams.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+
+Say you want to create a diagram showing only EC2 instances and RDS databases without the `Environment` tag. Click the **Resource** section, deselect all resource types, and select only `EC2` and `RDS`. Then click the **Custom tags** section, click the **Environment** tag, and leave only the `Untagged` option selected.
+
+{{< img src="cloudcraft/getting-started/crafting-better-diagrams/select-specific-resources-and-tags.mp4" alt="A 15 seconds video showing a Cloudcraft user selecting EC2 and RDS instances and untagged resources from the Resource and Custom tags sections." video="true">}}
+
+Click **Apply layout** to create a diagram showing only the selected resources without the `Environment` tag.
```
Comment (estherk15) [inline content/en/cloudcraft/getting-started/crafting-better-diagrams.md:70]:
```suggestion
### Create a diagram showing EC2 instances and RDS databases without the `Environment` tag

1. Click the **Resource** section.
1. Deselect all resource types, and select **EC2** and **RDS**.
1. Click the **Custom tags** section.
1. Click the **Environment** tag, and leave only the `Untagged` option selected.
1. Click **Apply layout** to create a diagram showing only the selected resources without the `Environment` tag.

{{< img src="cloudcraft/getting-started/crafting-better-diagrams/select-specific-resources-and-tags.mp4" alt="A 15 seconds video showing a Cloudcraft user selecting EC2 and RDS instances and untagged resources from the Resource and Custom tags sections." video="true">}}
```

File: content/en/cloudcraft/getting-started/use-filters-to-create-better-diagrams.md [hunk 8]
```diff
@@ -9,6 +9,8 @@ To avoid these issues, Cloudcraft recommends that you use the **Filtered layout*
 
 Building smaller diagrams makes managing them much easier. It also gives viewers more control over how they ingest information.
 
+<div class="alert alert-info">If you are using Cloudcraft's new live experience, please read this article instead: <a href="https://docs.datadoghq.com/cloudcraft/getting-started/crafting-better-diagrams/" title="Crafting Better Diagrams: Cloudcraft's Live Diagramming and Filtering">Crafting Better Diagrams: Cloudcraft's Live Diagramming and Filtering</a>.</div>
```
Comment (estherk15) [inline content/en/cloudcraft/getting-started/use-filters-to-create-better-diagrams.md:12]:
```suggestion
<div class="alert alert-info">If you are using Cloudcraft's New Live Experience, see this documentation: <a href="https://docs.datadoghq.com/cloudcraft/getting-started/crafting-better-diagrams/" title="Crafting Better Diagrams: Cloudcraft's Live Diagramming and Filtering">Crafting Better Diagrams: Cloudcraft's Live Diagramming and Filtering</a>.</div>
```

File: content/en/cloudcraft/getting-started/crafting-better-diagrams.md [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+Before you begin, you must connect your AWS or Azure account to Cloudcraft. For more information, see:
+
+- [Connect your AWS account with Cloudcraft][1]
+- [Connect your Azure account with Cloudcraft][2]
+
+## Enable new live experience
+
+To enable the new live experience, toggle the **New Live Experience** switch at the top of the **Live** tab in Cloudcraft.
+
+{{< img src="cloudcraft/getting-started/crafting-better-diagrams/enable-new-experience.png" alt="Screenshot highlighting the switch to enable the new live experience beta feature in the Cloudcraft interface with a red arrow pointing to the switch." responsive="true" style="width:100%;">}}
```
Comment (estherk15) [inline content/en/cloudcraft/getting-started/crafting-better-diagrams.md:25]:
```suggestion
{{< img src="cloudcraft/getting-started/crafting-better-diagrams/enable-new-experience.png" alt="Screenshot highlighting the switch to enable the new live experience beta feature in the Cloudcraft interface with a red arrow pointing to the switch." responsive="true" style="width:80%;">}}
```

**Review by estherk15 — CHANGES_REQUESTED** [review]
The closing tag for `nextlink` needs to match. Most suggestions are for formatting, let me know if have any questions!

**Comment by jamesponddotco** [comment]
@estherk15 Ready for a second review, then :)

File: content/en/cloudcraft/getting-started/crafting-better-diagrams.md [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+## Prerequisites
+
+Before you begin, you must connect your AWS or Azure account to Cloudcraft. For more information, see:
+
+- [Connect your AWS account with Cloudcraft][1]
+- [Connect your Azure account with Cloudcraft][2]
+
+## Enable new live experience
+
+To enable the new live experience, toggle the **New Live Experience** switch at the top of the **Live** tab in Cloudcraft.
```
Comment (estherk15) [inline content/en/cloudcraft/getting-started/crafting-better-diagrams.md:23]:
```suggestion
To enable, toggle the **New Live Experience** switch at the top of the **Live** tab in Cloudcraft.
```

File: content/en/cloudcraft/getting-started/crafting-better-diagrams.md [hunk 11]
```diff
@@ -0,0 +1,82 @@
...
+
+{{< img src="cloudcraft/getting-started/crafting-better-diagrams/enable-new-experience.png" alt="Screenshot highlighting the switch to enable the new live experience beta feature in the Cloudcraft interface with a red arrow pointing to the switch." responsive="true" style="width:80%;">}}
+
+If you're a new user, you may already have the new live experience enabled by default.
```
Comment (estherk15) [inline content/en/cloudcraft/getting-started/crafting-better-diagrams.md:27]:
```suggestion
If you're a new user, you may already have the New Live Experience enabled by default.
```

**Review by estherk15 — APPROVED** [review]

**Comment by jamesponddotco** [comment]
@estherk15 Yep, good to merge. Thank you!

**Pull request merged**